recipes/cuda-nvcc/nvcc.profile.patch

```diff
@@ -0,0 +1,14 @@
+--- bin/nvcc.profile	2023-03-23 15:56:49.535384866 -0400
++++ bin/nvcc.profile.new	2023-03-23 19:21:08.071151138 -0400
+@@ -5,9 +5,9 @@
+ 
+ PATH            += $(TOP)/$(_NVVM_BRANCH_)/bin;$(_HERE_);$(TOP)/lib;
+ 
+-INCLUDES        +=  "-I$(TOP)/include" $(_SPACE_)
++INCLUDES        +=  "-I$(TOP)/include" $(_SPACE_) "-I$(TOP)/include/targets/$(_WIN_PLATFORM_)" $(_SPACE_)

```

Comment:
Where is `include/targets/` used? I know we have `/targets/arch/include`

Reply:
We put the `cuda-cccl` headers under `include/targets/$(_WIN_PLATFORM_)`. Windows doesn't typically use the `targets` layout, but we needed a safe space to not clobber the GitHub releases.

Reply:
Gotcha, thanks for the explanation.